etc/config/php.amazon.properties

```diff
@@ -0,0 +1,17 @@
+# Default settings for PHP
+compilers=&php7
+defaultCompiler=php720
+
+group.php7.compilers=phpmaster:php720:php7112:php7026
+compiler.phpmaster.exe=/opt/compiler-explorer/php-master/bin/php
+compiler.phpmaster.alias=/usr/bin/php-master
+compiler.phpmaster.name=php master
```

Comment:
php name should probably be uppercase to match the rest of the languages

Also, this specifical version, to match the rest of the compilers versionings, should get the `master` changed to `(trunk)`